Userland/Libraries/LibPDF/Fonts/CFF.cpp

```diff
@@ -138,105 +138,247 @@ PDFErrorOr<NonnullRefPtr<CFF>> CFF::create(ReadonlyBytes const& cff_bytes, RefPt
     return cff;
 }
 
-HashMap<CFF::SID, DeprecatedFlyString> CFF::builtin_names {
-    { 0, ".notdef" },
-    { 1, "space" },
-    { 9, "parenleft" },
-    { 10, "parenright" },
-    { 13, "comma" },
-    { 14, "hyphen" },
-    { 15, "period" },
-
-    { 17, "zero" },
-    { 18, "one" },
-    { 19, "two" },
-    { 20, "three" },
-    { 21, "four" },
-    { 22, "five" },
-    { 23, "six" },
-    { 24, "seven" },
-    { 25, "eight" },
-    { 26, "nine" },
-    { 27, "colon" },
-    { 28, "semicolon" },
-
-    { 34, "A" },
-    { 35, "B" },
-    { 36, "C" },
-    { 37, "D" },
-    { 38, "E" },
-    { 39, "F" },
-    { 40, "G" },
-    { 41, "H" },
-    { 42, "I" },
-    { 43, "J" },
-    { 44, "K" },
-    { 45, "L" },
-    { 46, "M" },
-    { 47, "N" },
-    { 48, "O" },
-    { 49, "P" },
-    { 50, "Q" },
-    { 51, "R" },
-    { 52, "S" },
-    { 53, "T" },
-    { 54, "U" },
-    { 55, "V" },
-    { 56, "W" },
-    { 57, "X" },
-    { 58, "Y" },
-    { 59, "Z" },
-    { 66, "a" },
-    { 67, "b" },
-    { 68, "c" },
-    { 69, "d" },
-    { 70, "e" },
-    { 71, "f" },
-    { 72, "g" },
-    { 73, "h" },
-    { 74, "i" },
-    { 75, "j" },
-    { 76, "k" },
-    { 77, "l" },
-    { 78, "m" },
-    { 79, "n" },
-    { 80, "o" },
-    { 81, "p" },
-    { 82, "q" },
-    { 83, "r" },
-    { 84, "s" },
-    { 85, "t" },
-    { 86, "u" },
-    { 87, "v" },
-    { 88, "w" },
-    { 89, "x" },
-    { 90, "y" },
-    { 91, "z" },
-
-    { 104, "quotesingle" },
-    { 105, "quotedblleft" },
-
-    { 111, "endash" },
-
-    { 116, "bullet" },
-
-    { 119, "quotedblright" },
-
-    { 137, "emdash" },
-
-    { 170, "copyright" },
+/// Appendix C: Predefined Charsets
+Array s_cff_builtin_names {
```

Comment:
```suggestion
static constexpr Array s_cff_builtin_names {
```
One final little nit make this `static constexpr` (this should not be r/w or visible to other files)

Reply:
Of course -- it was static before in the header file, and I meant to keep it like that, but forgot to carry over the `static` modifier from the header file.